boundaries/build/index.json

```diff
@@ -1,2 +1,17 @@
-{}
+[
+  {
+    "directory": "country",
+    "area_type_wikidata_item_id": "Q6256",
+    "associations": [
+      {
+        "comment": "President of Mexico",
+        "position_item_id": "Q628004"
+      }
+    ],
+    "name_columns": {
+      "lang:en_US": "name_en",
```

Comment:
The language codes should me `lang:en` and `lang:es` (as of about yesterday, sorry).